neo/rawio/neuralynxrawio/nlxheader.py

```diff
@@ -223,6 +229,8 @@ def __init__(self, filename):
                 hpd = NlxHeader.header_pattern_dicts['bv5']
             elif av <= Version('5.4.0'):
                 hpd = NlxHeader.header_pattern_dicts['v5.4.0']
+            elif av == Version('5.6.0'):
```

Comment:
@PeterNSteinmetz Do you have any idea if this style was also used in other versions between 5.4.0 and 5.6.4?

Reply:
Unfortunately no. The manufacturer never documented these changes and has not responded to requests for better clarification. Thus we are stuck with observing changes as they come in.

Reply:
Ok, then we have to keep guessing based on the data files we come across :/